src/img/worker.py

```diff
@@ -146,6 +146,7 @@ def __init__(self, vm_kernel, ssh_key=None, log_filename=None,
                     '-display', 'none', '-no-reboot',
                     '-daemonize', '-m', '1G',
                     '-smp', '2',
+                    '-device', virtio-rng-pci,
```

Comment:
shouldn't this be '-device', 'virtio-rng-pci' ?

Reply:
yeah, will fix